dev-db/mariadb-connector-c/files/mariadb-connector-c-3.3.4-remove-zstd.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/plugins/compress/CMakeLists.txt b/plugins/compress/CMakeLists.txt
```

Comment:
Can you put a comment here explaining it (why we do it, i.e. that it conflicts w/ mariadb)?

Reply:
Sure, comment added.